### PR TITLE
Contact page ES6 conversion

### PR DESCRIPTION
**Pull request by jelliotartz**
## Describe your changes
- Staging branch URL: https://contact-page-es6-conversion.pfe-preview.zooniverse.org/about/contact
- Converts Contact component from coffeescript to ES6.
- Adds lifecycle hooks to add & remove `.on-secondary-page`.
- Adds unit tests for Contact component.
- Moves text to `locales/en`.
- Towards #4187.

### Required Manual Testing

- [x] Does the non-logged in home page render correctly?
- [x] Does the logged in home page render correctly?
- [x] Does the projects page render correctly?
- [x] Can you load project home pages?
- [x] Can you load the classification page?
- [x] Can you submit a classification?
- [x] Does talk load correctly?
- [x] Can you post a talk comment?

### Review Checklist

- [x] Does it work in all major browsers: Firefox, Chrome, Edge, Safari?
- [x] Does it work on mobile?
- [x] Can you `rm -rf node_modules/ && npm install` and app works as expected?
- [x] If the component is in coffeescript, is it converted to ES6? Is it free of eslint errors? Is the conversion its own commit?
- [x] Are the tests passing locally and on Travis?

### Optional

- [x] Have you replaced any `ChangeListener` or `PromiseRenderer` components with code that updates component state?
- [x] If changes are made to the classifier, does the dev classifier still work?
- [x] Have you [resized and compressed](https://developers.google.com/web/fundamentals/performance/optimizing-content-efficiency/image-optimization) any image you've added?
- [ ] Have you added in [flow type annotations](https://flowtype.org/docs/type-annotations.html)?
- [x] Have you followed the [Springer guidelines for commit messages](https://github.com/springernature/frontend-playbook/blob/master/git/git.md#commit-messages)?

### CODE REVIEW

File: app/pages/about/contact.spec.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+/* eslint prefer-arrow-callback: 0, func-names: 0, 'react/jsx-filename-extension': [1, { "extensions": [".js", ".jsx"] }] */
+
+import React from 'react';
+import assert from 'assert';
+import { shallow } from 'enzyme';
+import Contact from './contact';
+
+describe('Contact', function () {
+  it('renders without crashing', function () {
```
Comment (simoneduca) [inline app/pages/about/contact.spec.js]: @jelliotartz this test is empty.

Comment (simoneduca) [reply app/pages/about/contact.spec.js]: Perhaps something like `shallow(<Contact />);` would do.

Comment (jelliotartz) [reply app/pages/about/contact.spec.js]: This is an empty test per suggestion from Roger in [this PR](https://github.com/zooniverse/Panoptes-Front-End/pull/4190). Look down towards the bottom of the PR, right before the commit that changes the test to an empty function, for a hidden comment that you need to open up with the show/hide outdated tab.

Comment (mcbouslog) [reply app/pages/about/contact.spec.js]: In the case of AboutHome there was a before block that shallow rendered the component though, so I think Simone's suggestion is valid.

Comment (jelliotartz) [reply app/pages/about/contact.spec.js]: oops, duh, thank you! you're totally right!

**Pull request merged**